applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -152,7 +139,9 @@ def __build_cmd_get(cores: Core, device: AudioDevice, build: BuildType,
     if os.name == 'nt':
         release_flag = release_flag.replace('\\', '/')
     if pristine:
-        build_cmd += " -p"
+        build_cmd += " --pristine=auto"
+
+    dest_folder = TARGET_AUDIO_FOLDER / "build" / options.transport / device / core / build
```

Comment:
Is "build" required as part of the path? I thought ./build/ would be the top folder? (Within the audio application folder).

Reply:
After discussion, we can put the buildprog builds in the buildprog folder.